apps/nsq_to_nsq/nsq_to_nsq.go

```diff
@@ -380,6 +380,9 @@ func main() {
 		perAddressStatus: perAddressStatus,
 		timermetrics:     timer_metrics.NewTimerMetrics(*statusEvery, "[aggregate]:"),
 	}
+	if *mode == "epsilon-greedy" {
+		handler.hostPool = hostpool.NewEpsilonGreedy(destNsqdTCPAddrs, 0, &hostpool.LinearEpsilonValueCalculator{})
+	}
```

Comment:
actually, now that I'm looking at the surrounding code - I think this could be cleaner if we move this block above the instantiation of `PublishHandler`, pivot on `*mode`, assign a local `hostPool` variable, and reference that when instantiating.